[CTOR-1721]-Refresh applications-databases-firebird doc page (#4519)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-databases-firebird.md

|:----------------|:---------------------------------------|:----------------------------------------|
| Connection-Time | App-DB-Firebird-Connection-Time-custom | Check the connection time to the server |
| Memory          | App-DB-Firebird-Memory-custom          | Check memory usage                      |
| Pages           | App-DB-Firebird-Pages-custom           | Check page statistics                   |
| Queries         | App-DB-Firebird-Queries-custom         | Check queries statistics                |
| Users           | App-DB-Firebird-Users-custom           | Check current users connected           |

> The services listed above are created automatically when the **App-DB-Firebird-custom** host template is used.

</TabItem>
<TabItem value="Not attached to a host template" label="Not attached to a host template">

| Service Alias | Service Template                    | Service Description                  |
|:--------------|:------------------------------------|:-------------------------------------|
| Long-Queries  | App-DB-Firebird-Long-Queries-custom | Check current number of long queries |

> | Sql-Generic   | App-DB-Firebird-Sql-Generic-custom  | Check using a custom SQL query |

Reviewer: ```suggestion
| Sql-Generic   | App-DB-Firebird-Sql-Generic-custom  | Check allowing to perform a custom SQL query |
```

> | Sql-Generic   | App-DB-Firebird-Sql-Generic-custom  | Check using a custom SQL request     |

Reviewer 2: ```suggestion
| Sql-Generic   | App-DB-Firebird-Sql-Generic-custom  | Check using a custom SQL query |
```

> | Sql-Generic   | App-DB-Firebird-Sql-Generic-custom  | Check using a custom SQL request     |

Reviewer 2: unclear

| pages.marks.persecond   | /s    |

> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.

</TabItem>
<TabItem value="Queries" label="Queries">

| Name                              | Unit  |
|:----------------------------------|:------|
| queries.total.persecond           | /s    |
| queries.sequentialreads.persecond | /s    |
| queries.insert.persecond          | /s    |
| queries.updates.persecond         | /s    |
| queries.deletes.persecond         | /s    |
| queries.backout.persecond         | /s    |
| queries.purges.persecond          | /s    |
| queries.expunges.persecond        | /s    |

> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.

</TabItem>
<TabItem value="Sql-Generic" label="Sql-Generic">

| Name                              | Unit  |
|:----------------------------------|:------|
| value                             | N/A   |
| sqlrequest.execution.time.seconds | s     |

</TabItem>
<TabItem value="Users" label="Users">

| Name        | Unit  |
|:------------|:------|
| users.count | count |

</TabItem>
</Tabs>

## Prerequisites

> The Firebird server must be installed, running, and reachable from the Centreon poller

Reviewer 2: ```suggestion
The Firebird server must be installed, running, and reachable from the Centreon poller.
```
